### PR TITLE
chore(docs): don't use bower for docs dependencies

### PR DESCRIPTION
**Pull request by petebacondarwin**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

chore

**What is the current behavior? (You can also link to an open issue here)**

Docs app uses bower

**What is the new behavior (if this is a feature change)?**

No use of bower in the docs application

**Does this PR introduce a breaking change?**

No

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Other information**:

### CODE REVIEW

File: docs/config/services/deployments/debug.js [hunk 1]
```diff
@@ -17,9 +17,9 @@ module.exports = function debugDeployment(getVersion) {
       '../angular-sanitize.js',
       '../angular-touch.js',
       '../angular-animate.js',
-      'components/marked-' + getVersion('marked', 'node_modules', 'package.json') + '/lib/marked.js',
+      'components/marked-' + getVersion('marked') + '/lib/marked.js',
```
Comment (Narretz) [inline docs/config/services/deployments/debug.js]: Are these changes (also in the other deploy files) related to the switch to bower?

**Comment by Narretz** [comment]
Looks pretty straightforward. Did you test that the docs app works as expected with the library updates?

**Review by gkalpak — APPROVED** [review]
LGTM

**Comment by petebacondarwin** [comment]
@Narretz - yes I tested it manually and also we have a small handful of e2e tests for the docs app.

**Comment by Narretz** [comment]
@petebacondarwin LGTM

**Pull request merged**